lib/blockchain.py

```diff
@@ -433,13 +467,31 @@ def get_bits(self, header, chunk=None):
             raise Exception("get_bits missing header {} with chunk {!r}".format(height - 1, chunk))
         bits = prior['bits']
 
-        #NOV 13 HF DAA
+        # NOV 13 HF DAA and/or ASERT DAA
 
-        prevheight = height -1
+        prevheight = height - 1
         daa_mtp = self.get_median_time_past(prevheight, chunk)
 
-        #if (daa_mtp >= 1509559291):  #leave this here for testing
-        if (daa_mtp >= 1510600000):
+
+        # ASERTi3-2d DAA activated on Nov. 15th 2020 HF
+        if daa_mtp >= networks.net.asert_daa.MTP_ACTIVATION_TIME:
+            header_ts = header['timestamp']
+            prev_ts = prior['timestamp']
+            if networks.net.TESTNET:
+                # testnet 20 minute rule
+                if header_ts - prev_ts > 20*60:
+                    return MAX_BITS
+
+            anchor = self.get_asert_anchor(prior, daa_mtp, chunk)
+            assert anchor is not None, "Failed to find ASERT anchor block for chain {!r}".format(self)
```

Comment:
Maybe better to have exception thrown by `get_asert_anchor` itself?

Reply:
Eh.. 6 of 1 half dozen of the other, no?